package.json

```diff
@@ -91,6 +91,7 @@
     "@babel/preset-react": "^7.10.4",
     "@babel/preset-typescript": "^7.10.4",
     "@babel/runtime": "^7.9.2",
+    "@hanreev/types-ol": "^3.4.2",
```

Comment:
Is this change really what we want? I'd assume the original ol types are what we want to use

Reply:
The `@hanreev/types-ol` are an extension of the original ol types. It just offers additional type security.

Reply:
Yes, I thought if it is used in this way at `react-geo`, we should adapt it here as well. 
If there are no objections, I will merge these changes soon.